### PR TITLE
fix(card-pictogram): fixed visual bugs & storybook refinements

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)
#5557

### Description
Removed unnecessary Story knobs in `Card` and `Card With Pictogram`, as well as fixing the pictogram size.

### Changelog

**Changed**

- ensured pictogram takes up 48x48 container

**Removed**

- unnecessary Storybook knobs
- eyebrow use in `Card with pictogram`

### CODE REVIEW

**Review by shixiedesign — COMMENTED** [review]
Hi Ignacio! So glad to see these updates! Here are a few bugs I caught: 
1. There should always be a spacing of 32px under the pictogram to give it some room to breath. In this case, too much text is squeezing against it.
image
2. When picto bot, missing spacing 32px above pictogram too.
3. When picto bot, the hover increases the height of card. This is a bug and could cause issues when the card is in a group... if copy is wrapping to be taller than 48px, let's truncate for now, and add `...`. It's problematic I know and we will look into it in an enhancement issue. 

Apr-08-2021 12-47-43

**Comment by IgnacioBecerra** [comment]
FYI, In order to force a truncation at 2 lines, I'm using [-webkit-line-clamp](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp), which has widespread support across browsers ([Caniuse](https://caniuse.com/mdn-css_properties_-webkit-line-clamp))

**Comment by shixiedesign** [comment]
Hey Ignacio! Thank you so much and it's almost there - 2 problems and both captured in this gif: 

- Let's truncate after 3 lines. I think 2 lines might not be sufficient for many situ's content need and it still looks roomy enough.
- Focus state would need to keep that 3 line truncation so the card doesn't grow

Apr-09-2021 23-40-21

**Review by shixiedesign — APPROVED** [review]
I did notice some potential issues for future if the title wraps to 2 lines... but I think this is already a huge improvement and we will get to revisit! 

PS: When title wraps to 2 lines or more, hover makes the card grew again. I will create a new ticket to address this as a design issue. Chances are this hover interaction is not thought through. Thank you!

File: packages/web-components/src/components/card/card.scss [hunk 1]
```diff
@@ -75,8 +54,28 @@
...
+    &:focus {
+      ::slotted(svg[slot='pictogram']) {
+        display: none;
+      }
+
+      /* stylelint-disable value-no-vendor-prefix, property-no-vendor-prefix */
+      .#{$prefix}--card__copy {
+        display: -webkit-box;
+        -webkit-box-orient: vertical;
+        -webkit-line-clamp: 3;
```
[the remark this answers is not in the document]
Comment (IgnacioBecerra) [reply packages/web-components/src/components/card/card.scss:67]: Just tested it out on Firefox and Safari, and can confirm it looks the same. I also checked on caniuse.com, and it seems there's widespread browser support for this. 
https://caniuse.com/mdn-css_properties_-webkit-line-clamp

**Pull request merged**